geometry.pyi

```diff
@@ -92,6 +92,7 @@ class Line(Sequence[float]):
     @overload
     def collideline(self, first: Sequence[float], second: Sequence[float]) -> bool: ...
     def colliderect(self, rect: RectValue) -> bool: ...
+    def collideswith(self, other: _CanBeCollided) -> bool: ...
```

Comment:
This is between two `colliderect()` functions, could you move them somewhere else?